### ISSUE TITLE
Pie Chart bug?

### ISSUE THREAD

**Issue by ryanhamilton** [timestored/pulseui#171]
user-ag reports colleague can create multi-pie in qStudio
But same query in Pulse doesn't work.

**Comment by ryanhamilton** [comment]
Select and anything above 4 pies in Pulse doesn't work.
Problem is that dynamic sizing in pulse is much more difficult than qStudio. 
qStudio makes everything small proportionally.
Pulse, the shrinking of title text isn't proportional.


```
t:([] s:`p`o`i; a:1 2 3; b:9 6 2; c:8 5 3; d:8 7 2; e:9 8 7; f:2 9 4; g:9 4 7; h:1 1 7; j:9 9 3; k:9 8 9)
select a from t
(2#cols t)#t
(3#cols t)#t
(4#cols t)#t
(5#cols t)#t
(6#cols t)#t
(7#cols t)#t
(8#cols t)#t
(9#cols t)#t
``s _ (10#cols t)#t
```

![image](https://github.com/timestored/pulseui/assets/7273208/806db39a-355e-43b5-befa-92d764ff4eb0)

**Comment by ryanhamilton** [comment]
Dynamic spacing:

![image](https://github.com/timestored/pulseui/assets/7273208/9564d28a-a0b1-4b3f-a689-c91ffbb71173)

![image](https://github.com/timestored/pulseui/assets/7273208/303ca410-1187-4417-bff9-267a579b60eb)

![image](https://github.com/timestored/pulseui/assets/7273208/636eae75-b0b3-48c6-886f-5123cf269e69)

**Comment by ryanhamilton** [comment]
Improved handling of single row pie chart to be more useful (slice per column, rather than pie per column).
Also same as qStudio.

**Comment by ryanhamilton** [comment]
Released 2.23

![image](https://github.com/timestored/pulseui/assets/7273208/8b8a1986-a89e-4e6e-a177-4109add33319)
